dbfit-java/core/src/main/java/dbfit/api/DBEnvironment.java

```diff
@@ -189,4 +189,6 @@ void connectUsingFile(String filePath) throws SQLException, IOException,
             FileNotFoundException;
 
     DbStoredProcedureCall newStoredProcedureCall(String name, DbParameterAccessor[] accessors);
+
+    String buildFunctionCall(String procOrFuncName, int numberOfParameters);
```

Comment:
For stored procedures we already have `DbStoredProcedureCall newStoredProcedureCall(String name, DbParameterAccessor[] accessors)` which should be fulfilled by the environment.

Let's think about what we're unhappy about it? (Some points are maybe already touched in #463 - e.g. like returning type is *too much*, but it doesn't hurt to take a fresh look at it).